packages/compass-connections/src/stores/connections-store.ts

```diff
@@ -221,7 +221,7 @@ async function loadConnections(
   connectionStorage: ConnectionStorage
 ) {
   try {
-    const loadedConnections = await connectionStorage.loadAll();
+    const loadedConnections = await connectionStorage.loadAll({});
```

Comment:
I don't think passing empty objects here is needed, at least the types should handle that (and if they are not, we probably should look a bit into that, I think for atlas-service this was working)

Reply:
fixed.